src/main/java/TaskListener.java

```diff
@@ -0,0 +1,191 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class TaskListener {
+    static final String DIVIDER = "--------------------------------------------------------------------";
+    private TaskList taskList;
```

Comment:
While TaskList is a custom object, it still is primarily a container of tasks, and so should follow the naming convention of having a plural form like `tasksList`.

Reply:
Thanks for a suggestion! Will fix that